.github/workflows/format.yml

```diff
@@ -7,6 +7,8 @@ on:
   workflow_dispatch:
   pull_request:
   push:
+    branches:
+      - main
```

Comment:
Should this include other branches as well, for instance `feature/hybrid-planning` after that one has been rebased?

Reply:
I didn't know there were other branches.  I'll add that.